spec/index.bs

```diff
@@ -4264,9 +4264,17 @@ run the following steps:
 
                 1. Let |textureLayout| be a new {{GPUTextureBindingLayout}}.
 
-                1. Set |textureLayout|.{{GPUTextureBindingLayout/sampleType}} to |resource|'s component type.
-
-                    Issue: Detail how the sampleType is chosen from the binding.
+                1. If |resource| is a depth texture binding:
+                    1. Set |textureLayout|.{{GPUTextureBindingLayout/sampleType}} to {{GPUTextureSampleType/"depth"}}
+                1. Else if the sampled type of |resource| is `f32`, and |resource| is statically used with a
```

Comment:
can we use a switch construct for the sample type?

Reply:
Done